docs/supported_apis/dataframe_supported.rst

```diff
@@ -279,11 +300,11 @@ default to pandas.
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
 | ``query``                  | `query`_                  | P                      | Local variables not yet supported                  |
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
-| ``radd``                   | `radd`_                   | Y                      | See ``add``                                        |
+| ``radd``                   | `radd`_                   | Y                      | See ``add``; **Omnisci**:``N``                     |
```

Comment:
As long `add` is supported `radd` is supported as well, so it should be "Y"
https://github.com/modin-project/modin/blob/5440f08c1dca02f12ea4a4479e6d3ea91cef5a66/modin/pandas/base.py#L1871

Reply:
Fixed